Bootcamp.sln

```diff
@@ -4,6 +4,9 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 VisualStudioVersion = 16.0.29519.87
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Src", "Src", "{3E67C921-D341-42FB-9934-B03C05A28BEB}"
+	ProjectSection(SolutionItems) = preProject
```

Comment:
I don't think it's needed

Reply:
Where did you find this file? It just states it is within Bootcamp.sln which, as far as I know, is the entire project.

Reply:
It's .sln file.

Reply:
So all you have to do is to edit the sln file and remove the changed lines.

Reply:
The only way I am aware I can work with the sln file is by opening it then browsing contents in the source explorer, which consists of the "Src" and "Tests" folders. Can you explain where I find the text above and edit it?